add_to_aosp.sh

```diff
@@ -10,13 +10,17 @@ if [ ! -d "$AOSP" ]; then
     printf "Directory %s does not exists" "$AOSP"
 fi
 
-echo "Overriting build/target"
-rsync -a --inplace "$GIT_ROOT"/build/target "$AOSP"/build/make
+echo "Overriting target"
```

Comment:
The typo was already there 🫨

```suggestion
echo "Overwriting target"
```